app/components/message_attachments/attachment_image.js

```diff
@@ -37,7 +37,7 @@ export default class AttachmentImage extends PureComponent {
         const {imageUrl, metadata} = this.props;
 
         this.setViewPortMaxWidth();
-        if (metadata?.images[imageUrl]) {
+        if (metadata?.images && metadata.images[imageUrl]) {
```

Comment:
Can this be written as `metadata?.images?[imageUrl]` ? This seems like a good way to use Optional Chaining

Reply:
@sudheerDev sadly your suggestion returns a parse error, as it does not work with [object-key] variables

Reply:
Could be `metadata?.images?.[imageUrl]`. Can you try this once.

Reply:
there you go